examples/one_dot_one/src/generatedNoCheck/objects/ObjectTypeWithAllPropertyTypes.ts

```diff
@@ -56,118 +56,147 @@ export const ObjectTypeWithAllPropertyTypes = {
     id: {
       multiplicity: false,
       type: 'integer',
+      nullable: true,
```

Comment:
Did you mean to make all of these nullable?

Reply:
Yeah, we don't have this metadata at the moment so they should all be nullable